<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Regresion_Lineal</class>
 <widget class="QMainWindow" name="Regresion_Lineal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;REGRESIÃ“N LINEAL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TEXTO_RL">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="EJECUTAR_RL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>EJECUTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BORRAR_RL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>BORRAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SALIR_RL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;SALIR</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x1_RL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x2_RL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x3_RL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x4_RL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x5_RL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x6_RL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x7_RL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y1_RL">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y2_RL">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y3_RL">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y4_RL">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y5_RL">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y6_RL">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y7_RL">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;x&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;y&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
